index.html

```diff
@@ -3000,69 +3000,72 @@ <h3>Lifecycle Details</h3>
         <h3>Trust Model</h3>
 
         <p>
-The [=verifiable credentials=] trust model is as follows:
+The [=verifiable credentials=] trust model is based on the following
+expectations:
         </p>
 
         <ul>
           <li>
-The [=verifier=] trusts the [=issuer=] to issue the [=credential=] that
-it received. To establish this trust, a [=credential=] is expected to either:
+The [=verifier=] expects the [=issuer=] to verifiably issue the
+[=credential=] that it receives. This can be established by satisfying
+either of the following:
             <ul>
               <li>
-Secure the [=credential=] with a <a href="#securing-mechanisms">securing mechanism</a> establishing that the
-[=issuer=] generated the [=credential=] (that is, it is a
-[=verifiable credential=]), or
+A [=verifier=] is expected to secure a [=credential=] with a 
```

Comment:
I think this one as well?
```suggestion
An [=issuer=] is expected to secure a [=credential=] with a 
```